### PR TITLE
Removing obsolete code to build documents

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Removing some old stuff that built documentation, back in the days when documentation was something that was built (py 2.7 days). I did try to revive the documentation builder but there were still errors after updating the command to py 3, and rendering the docs as html is not particularly useful when we have them as markdown.

# Jira Ticket / Issue #

N/A

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (Python) I have run Black and Flake8 on changed Python code manually or with a git precommit hook

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,9 @@ build-narrative-container:
 
 docker_image: build-narrative-container
 
+docker-narrative:
```
Comment (ialarmedalien) [inline Makefile:19]: moved up from the bottom of the file

File: Makefile [hunk 2]
```diff
@@ -28,7 +31,7 @@ install:
 
 # runs the installer to locally build the Narrative in a
 # local venv.
-build-travis-narrative:
+build-venv-narrative:
```
Comment (ialarmedalien) [inline Makefile:34]: we don't use travis any more

File: Makefile [hunk 3]
```diff
@@ -65,13 +68,5 @@ test-integration:
 	python scripts/run_tests.py -i
 	@echo "done"
 
-build-docs:
```
Comment (ialarmedalien) [inline Makefile:68]: ancient cruft

File: src/biokbase/narrative/common/util.py [hunk 4]
```diff
@@ -1,17 +1,19 @@
-"""
-Utility functions for dealing with KBase services, etc.
-"""
+"""Utility functions for dealing with KBase services, etc."""
```
Comment (ialarmedalien) [inline src/biokbase/narrative/common/util.py:1]: Made a few other changes to this file to make flake8 /  ruff happy.

File: src/biokbase/narrative/common/util.py [hunk 5]
```diff
@@ -91,24 +94,3 @@ def _user(self):
 # Get/set KBase environment variables by getting/setting
 # attributes of this object.
 kbase_env = _KBaseEnv()
-
-
-class BuildDocumentation(Command):
```
Comment (ialarmedalien) [inline src/biokbase/narrative/common/util.py:96]: old BuildDocumentation class. It is only referenced in the setup file (see below)

**Review by briehl — APPROVED** [review]
Goodbye old code!

**Pull request merged**